### PR TITLE
fix receiver cannot create new room

### PR DESCRIPTION
**Pull request by jilv220**
closed issue #97

### CODE REVIEW

File: src/components/StreamerConnection.vue [hunk 1]
```diff
@@ -59,6 +59,12 @@ export default {
       false,
     );
   },
+  onDestory() {
```
Comment (codysherman) [inline src/components/StreamerConnection.vue]: Destroy is spelled wrong, and try using `beforeDestroy` instead.

File: src/components/StreamerConnection.vue [hunk 2]
```diff
@@ -59,6 +59,12 @@ export default {
...
+
+    clearInterval(this.connection.looper);
```
Comment (samtheprogram) [inline src/components/StreamerConnection.vue]: pretty sure `this.connection.looper` is undefined (the looper in this file is not defined on connection, but is just a local variable), buuuut....

File: src/components/StreamerConnection.vue [hunk 3]
```diff
@@ -59,6 +59,12 @@ export default {
...
+    this.connection = null;
```
Comment (samtheprogram) [inline src/components/StreamerConnection.vue]: .....this line is probably what was fixing the issue (prior to the `onDestory` typo :))

**Comment by codysherman** [comment]
This change is great and has made it live, but this PR was bundled into another for some minor fixes. So I'm closing this specific PR in favor of https://github.com/codysherman/2nfm/pull/129

**Comment by samtheprogram** [comment]
Fixed by #129 & #130

**Pull request closed without merging**